flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
maybe all of these flow changes should have been in a separate PR. Just a comment for next time, I'm not saying you should change it